ParquetClassLibrary/AssemblyInfo.cs

```diff
@@ -11,94 +11,177 @@
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test Items.
         /// </summary>
-        public static readonly List<Range<EntityID>> ParquetIDs = new List<Range<EntityID>>
+        public static readonly Range<EntityID> CharacterIDs;
+
+        /// <summary>
+        /// A subset of the values of <see cref="EntityID"/> set aside for <see cref="Characters.Player"/>s.
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test Items.
+        /// </summary>
+        public static readonly Range<EntityID> PlayerIDs;
+
+        /// <summary>
+        /// A collection containing all defined <see cref="Range{EntityID}"/>s of <see cref="Characters.Being"/>s.
+        /// </summary>
+        public static readonly List<Range<EntityID>> BeingIDs = new List<Range<EntityID>>
             {
-                FloorIDs, BlockIDs, FurnishingIDs, CollectibleIDs
+                CritterIDs, CharacterIDs, PlayerIDs
             };
+
+        /// <summary>
+        /// A subset of the values of <see cref="EntityID"/> set aside for <see cref="Sandbox.RoomRecipe"/>s.
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test Items.
+        /// </summary>
+        public static readonly Range<EntityID> RoomRecipeIDs;
+
+        /// <summary>
+        /// A subset of the values of <see cref="EntityID"/> set aside for <see cref="Crafting.CraftingRecipe"/>s.
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test Items.
+        /// </summary>
+        public static readonly Range<EntityID> CraftingRecipeIDs;
+
+        /// <summary>
+        /// A subset of the values of <see cref="EntityID"/> set aside for <see cref="Quests.Quest"/>s.
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test Items.
+        /// </summary>
+        public static readonly Range<EntityID> QuestIDs;
+
+        /// <summary>
+        /// A subset of the values of <see cref="EntityID"/> set aside for <see cref="Items.Item"/>s.
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test Items.
+        /// </summary>
+        public static readonly Range<EntityID> ItemIDs;
+        #endregion
+
+        #region Initialization
+        /// <summary>
+        /// Initializes the <see cref="Range{EntityID}"/>s defined in <see cref="AssemblyInfo"/>.
+        /// This supports defining ItemIDs in terms of the other Ranges.
+        /// </summary>
+        static AssemblyInfo()
+        {
+            // By convention, the first EntityID in each Range is a multiple of this number.
+            // An exception is made for Critter, Character, and Player as they are treated as sub-ranges.
+            var TargetMultiple = 10000;
+
+            FloorIDs = new Range<EntityID>(10000, 19000);
```

Comment:
TargetMultiple and these numbers line up fairly well. Perhaps it can be used here to reduce magic numbering?

CritterIDs, CharacterIDs, and PlayerIDs don't follow the same convenient convention as others, so I can see why you wouldn't change this.

Reply:
I thought about this but I felt like it would be clearer in this case to include the magic numbers when referencing the code while designing.  I will keep thinking about this.